### DISCUSSION TITLE
Welcome to newsletter Discussions!

### DISCUSSION THREAD

**Discussion by chavarera** [Python-World/newsletter#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by risingwthewind** [comment]
Learning to become the best at helping other developers with wisdom and understanding
